mod_reforged/config/msu_excluded_injuries.nut

```diff
@@ -34,6 +35,7 @@
 		"injury.pierced_side",
 		"injury.pierced_arm_muscles",
 		"injury.stabbed_guts",
+		"injury.rf_black_eye"
```

Comment:
Black Eye should be removed from RF_Undead instead of only from RF_Skeleton. Undead do not have a functioning blood circulatory system and therefore shouldn't be getting bruises or swollen body parts from getting hit.

Dislocated Jaw should also be removed from skeletons imo but we can discuss this in design discussion if you like.